web/themes/custom/contribtracker/package.json

```diff
@@ -45,6 +55,8 @@
     "uuid": "^10.0.0"
   },
   "dependencies": {
+    "cypress-split": "^1.24.0",
+    "@slack/web-api": "^6.11.2",
```

Comment:
Why are we using `@slack/web-api` package? Are we planning to send a notification to slack?

Reply:
@zeshanziya Yes, this is for functional tests notification. Likely to attach screenshots of failures

Reply:
@KosalaiV Let's remove it for now. For Slack integration, we can have another PR. Our intention is to keep it simple, so only changes related to this PR should be included. Any additional changes can be added later. Let's work with small PRs so it's easier to review and merge faster.